lib/iris/coords.py

```diff
@@ -1215,28 +1209,20 @@ def serialize(x):
                     'Metadata may not be fully descriptive for {!r}.'
```

Comment:
There is a warning above:

```
                 msg = 'Collapsing a multi-dimensional coordinate. ' \
                     'Metadata may not be fully descriptive for {!r}.'
```

I guess that triggers when we completely collapse a multi-dim coord (rather than partially).

Nothing to do here, just reminding myself of the code! 😄